src/notifications/notification.spec.ts

```diff
@@ -83,6 +83,58 @@ describe('`nglNotification`', () => {
     externalCloseButton.click();
   }));
 
+  it('should emit a close event when the 500ms timeout has passed', testAsync(({fixture, done}) => {
+    fixture.detectChanges();
+
+    spyOn(fixture.componentInstance, 'onClose').and.callFake((reason: string) => {
+      expect(reason).toBe('timeout');
+      done();
+    });
+
+    fixture.componentInstance.timeout = 500;
+    fixture.detectChanges();
+  }));
+
+  it('should cancel the 500ms timeout after 300ms', testAsync(({fixture, done}) => {
```

Comment:
Same as before, something like `should cancel previous timeout and reset if input changes` would be more appropriate.